tests/wdk-sys-tests/tests/nt_status_functions.rs

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) Microsoft Corporation
```

Comment:
Having this need to be a separate crate in order to run the test in an environment that has a WDK config is unfortunate. I have something in the works that will allow these unit tests to live in the same module of the functions they're testing

Reply:
tracked in #211